helm-charts/stable/amazon-eks-pod-identity-webhook/values.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+nameOverride: "pod-identity-webhook"
+fullnameOverride: "pod-identity-webhook"
+
+# Make sure to pass in the image for the webhook here or as a parameter to the helm installation
+image: placeholder-webhook:fake
+
+baseImage: public.ecr.aws/amazonlinux/amazonlinux:2 
```

Comment:
nit: can you add a doc line above this for what this is used for?

Reply:
Will do